src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/TensorPrimitives.netcore.cs

```diff
@@ -54,7 +54,6 @@ public static void ConvertToHalf(ReadOnlySpan<float> source, Span<Half> destinat
             ref ushort destinationRef = ref Unsafe.As<Half, ushort>(ref MemoryMarshal.GetReference(destination));
             int i = 0, twoVectorsFromEnd;
 
-#if NET8_0_OR_GREATER
```

Comment:
How much more are we able to "clean" this up in main?

Are we considering switching over to using `ISimdVector` and similar to reduce code duplication between the paths or are we only minimally differing so that backporting isn't problematic if still required?

Reply:
> Are we considering switching over to using ISimdVector and similar to reduce code duplication between the paths or are we only minimally differing so that backporting isn't problematic if still required?

ISimdVector is currently internal to corelib.